private/rules/kt_jvm_export.bzl

```diff
@@ -0,0 +1,134 @@
+load("@io_bazel_rules_kotlin//kotlin:jvm.bzl", "kt_jvm_library")
+load("//private/rules:javadoc.bzl", "javadoc")
+load("//private/rules:maven_project_jar.bzl", "maven_project_jar")
+load("//private/rules:maven_publish.bzl", "maven_publish")
+load("//private/rules:pom_file.bzl", "pom_file")
+
+def kt_jvm_export(
```

Comment:
This shares a significant chunk of code with java_export (and if there are differences other than the java_library/kt_jvm_library swap, it's hard to tell). Can you try to reuse the implementation across the two macros?

Reply:
Yep, I made a new maven_export macro that both java_export and kt_jvm_export can use.